types/chrome/chrome-cast/index.d.ts

```diff
@@ -766,13 +720,13 @@ declare namespace chrome.cast.media {
         currentTime: number;
         customData: Object;
         media: chrome.cast.media.MediaInfo;
+        playbackRate: number;
```

Comment:
```suggestion
        playbackRate: number | undefined;
```

as per https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.media.LoadRequest#playbackRate